### DISCUSSION TITLE
Question about compatibility of the new escape code parser

### DISCUSSION THREAD

**Discussion by solopasha** [kovidgoyal/kitty#7163] [General]
As changes from the vt branch have been merged recently, I have a question about compatibility and how distros should handle this. Will kitty work on older CPUs that don't support sse42? Some distributions don't allow packaging software that doesn't work on all x86-64 CPUs.

**Comment by kovidgoyal** [comment]
Yes, it will and does.

**Reply by solopasha** [reply]
Great, thanks for the clarification and for your work!
